introduction.md

```diff
@@ -1,2 +1,6 @@
 Hello, my username is patrickmelvin
 My favorite codeschool path is the Ruby path
+
+* Git
+* Ruby
+* Javascript
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?